requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Everything needed in production
-werkzeug<2.2.3
+werkzeug<2.3
```

Comment:
@dnil this should cover the situation with Docker

Reply:
Super; yes, and allow for stepping over 2.2.2 to make dependabot happy.